nixio/multi_tag.py

```diff
@@ -52,7 +52,15 @@ def positions(self, da):
             raise TypeError("MultiTag.positions cannot be None.")
         if "positions" in self._h5group:
             del self._h5group["positions"]
-        self._h5group.create_link(da, "positions")
+        pos = da
+        if not isinstance(da, DataArray):
+            blk = self._parent
+            name = "{}-positions".format(self.name)
+            if name in blk.data_arrays:
+                del blk.data_arrays[name]
+            pos = blk.create_data_array(name, "{}-positions".format(self.type),
+                                        data=da)
```

Comment:
What if `da` is an integer or string or some other invalid value?  I think it would be confusing for the user if they were trying to create a MultiTag and they got an error about DataArray creation with invalid data.  We should catch this issue as early as possible, preferably in the `create_multi_tag()` method in the `Block` class.

Also, I think this might not be the best location for the convenience functionality to be in general.  Having it here makes it possible to create DataArrays by assigning values to `mtag.positions`.  While this might sound useful, it also makes it more likely that a user tries to do something like:
```python
mtag.positions = [10, 11]
# ... later
mtag.positions = [10, 11, 12]
# or
mtag.positions += [13]
```
which will lead to all sorts of errors: Duplicate names for the DataArray creation, attempting to append a list to a DataArray, etc.

I think the convenience should only be in the `block.create_multi_tag()` method and nowhere else, and it should be documented clearly in the arguments in the docstring.  The type check for the `positions` argument in `create_multi_tag` should check for `DataArray` or any kind of `sequence`.